src/test/java/com/wikia/webdriver/elements/mercury/components/Search.java

```diff
@@ -86,4 +72,40 @@ public Search clickClearSearchButton() {
     return this;
   }
 
+  public boolean isInputFieldSearchIconVisible() {
+    try {
+      wait.forElementVisible(inputFieldSearchIcon);
+      return inputFieldSearchIcon.isDisplayed();
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)